### ISSUE TITLE
Question: why are functions used in translation files?

### ISSUE THREAD

**Issue by notramo** [deadhead420/anarchy-linux#446]
Why is this:
```bash
example_function(){
example_string="Example text."
}

example_function
```
instead of this:
```bash
example_string="Example text."
```

**Comment by notramo** [comment]
I'm planning to create an interactive translation tool for Arch Linux Anywhere. If these functions could be removed, creating the tool would be easier, because it could use only regex, and shouldn't track the function the variable is in.

**Comment by deadhead420** [comment]
Hey man. That wouldn't be an issue at all. A translation tool would be great. Its a pain in the ass to add translations to each file and keep up with them. The reason I made it into functions is so every variable isn't reloaded each time the language file is sourced. I pretty much made it so that after the initial source of the lang file every other source only loads the variables in the functions containing "var" (because those functions contain variables set after the installer begins.). I did this initially because I thought it might be faster but in reality it makes no difference. Feel free to remove the functions if needed.
